RecoLocalCalo/HcalRecAlgos/src/HcalDeterministicFit.cc

```diff
@@ -121,35 +114,18 @@ void HcalDeterministicFit::phase1Apply(const HBHEChannelInfo& channelData,
 
   const HcalDetId& cell = channelData.id();
 
-  double fpar0, fpar1, fpar2;
-  if(std::abs(cell.ieta())<HcalRegion[0]){
-    fpar0 = fpars[0];
-    fpar1 = fpars[1];
-    fpar2 = fpars[2];
-  }else if(std::abs(cell.ieta())==HcalRegion[0]||std::abs(cell.ieta())==HcalRegion[1]){
-    fpar0 = fpars[3];
-    fpar1 = fpars[4];
-    fpar2 = fpars[5];
-  }else{
-    fpar0 = fpars[6];
-    fpar1 = fpars[7];
-    fpar2 = fpars[8];
-  }
-
-  if (fTimeSlew==0)respCorr=1.0;
-  else if (fTimeSlew==1) channelData.hasTimeInfo()?respCorr=rCorrSiPM[0]:respCorr=rCorr[0];
-  else if (fTimeSlew==2) channelData.hasTimeInfo()?respCorr=rCorrSiPM[1]:respCorr=rCorr[1];
-  else if (fTimeSlew==3)respCorr=frespCorr;
+  HcalTimeSlew::ParaSource subDet=HcalTimeSlew::ParaSource::HE;
+  if(std::abs(cell.ieta())<HcalRegion[0]) subDet=HcalTimeSlew::ParaSource::HB;
+  else if(std::abs(cell.ieta())==HcalRegion[0]||std::abs(cell.ieta())==HcalRegion[1]) subDet=HcalTimeSlew::ParaSource::BE;
+  else subDet=HcalTimeSlew::ParaSource::HE;
```

Comment:
This line is not needed

Reply:
I thought it wasn't needed but it was setup before to give different time slew parameters for HB and HE separately (However, they gave the same numbers). 
I will remove this from here and from the HcalTimeSlew delay function.